src/useAudioPosition.ts

```diff
@@ -12,6 +12,7 @@ interface AudioPosition {
     duration: number
     percentComplete: number
     seek: (position: number) => number
+    speed: (rate?: number) => number | undefined
```

Comment:
hey @Xavier-Charles thanks for the PR. IMO this method would make more sense on the main hook rather the hook for playhead position

Reply:
Thanks to both @E-Kuerschner and @Xavier-Charles for your great efforts on this library!

I agree that this probably should rather be an option to the main hook. I also suggest using the same term as in howler, **rate**, instead of introducing **speed**?

Reply:
@phemartin apologies, i've been super busy with my full time. I'll try and get this added in soon!

Reply:
@phemartin @Xavier-Charles @aweibell just released a version 1.3.0 which exposes rate off of howler from the main useAudioPlayer hook. There is an example usage in examples folder (spotifyish). Closing this PR